<launch>
    <node pkg="point_cloud_io" type="read" name="read_stl_node" output="screen">
        <param name="file_path" value="$(find recon_surface)/test/map_medium.ply" />
        <param name="topic"     value="/test_point_cloud" />
        <param name="frame"     value="initial_base" />
        <param name="rate"      value="0.2" />
    </node>

    <node pkg="recon_surface" type="test_p_normal_service.py" name="test_p_normal_service" output="screen"></node>

</launch>